packages/babel-plugin-minify-constant-folding/src/replacements.js

```diff
@@ -43,11 +43,24 @@ module.exports = ({ types: t }) => {
           let bad = false;
           const str = this.elements
             .map(el => {
-              if (!t.isLiteral(el)) {
-                bad = true;
-                return;
+              if (t.isRegExpLiteral(el)) {
+                return `/${el.pattern}/${el.flags}`;
               }
-              return el.value;
+              if (t.isNullLiteral(el)) {
```

Comment:
IMO, we can do `path.evaluate(el)` and deopt. 

If its gonna be kill performance which in most cases should not be a problem since the path would already be cached during the traversal phase.. Thoughts?